pkg/services/ngalert/provisioning/contactpoints_test.go

```diff
@@ -493,6 +493,353 @@ func TestStitchReceivers(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "rename to another, larger group",
```

Comment:
This test was not affected by this change, it was just a case that I noticed we should cover.